src/pages/LandingPage.js

```diff
@@ -67,7 +71,39 @@ const makeDocLinks = _.map(
 )
 
 const LandingPage = () => {
+  const { isSignedIn } = useStore(authStore)
+  const [billingProjects, setBillingProjects] = useState()
+  const signal = useCancellation()
+
+  useEffect(() => {
+    const loadProjects = withErrorIgnoring(async () => {
```

Comment:
@yunip20 we can do the following to print a console error if `listProjects` fails (the stack trace and further details can be obtained then by looking at the result from the server call in the network tab):

````
useEffect(() => {
    const loadProjects = withErrorHandling( async(error) => {
      const errorJSON = await error.json()
      console.log(`Unable to load billing projects due to: ${errorJSON?.message}`)
    })
    (async () => {
      const projects = await Ajax(signal).Billing.listProjects()
      setBillingProjects(projects)
    })
    if (isSignedIn) {
      loadProjects()
    }
  }, [isSignedIn, setBillingProjects, signal])
  ```

 So for example, I get this with my account that has some billing accounts in an unexpected state:
  
![image](https://user-images.githubusercontent.com/484484/198076159-715e86d9-565e-4a62-a927-1238ccf5ffa7.png)